src/title-page.md

```diff
@@ -4,6 +4,10 @@ This book is a treasure-trove of Rust books in [mdbook](https://github.com/rust-
 * [Official books](./official.md) maintained at [rust-lang.org](https://www.rust-lang.org/)
 * [Unofficial books](./unofficial.md) maintained elsewhere
 * [Application books](./applications.md) on specific Rust applications
-* [Books in Other Formats](./other-formats.md)
 
 If you know of others, why not submit a [pull request](https://github.com/lborb/book#pull-requests)?
+
+Some books get an honourable mention even though they aren't actually in
```

Comment:
Can we remove use of word 'honourable' so that it reads 'Some books are mentioned even though..'. As well here I believe it is Rust Lang convention, as far as compiler internals and official docs are concerned, to use American English spelling, or do you prefer UK variants?

Reply:
Nice catch! I'm English, so prefer English spellings, but am happy to oblige. Let me rephrase and force push an update.